src/smc-hub/auth.coffee

```diff
@@ -221,7 +221,13 @@ passport_login = (opts) ->
             dbg("set remember_me cookies in client")
             expires = new Date(new Date().getTime() + ttl*1000)
             cookies = new Cookies(opts.req, opts.res)
-            cookies.set(BASE_URL + 'remember_me', remember_me_value, {expires:expires})
+            root_domain = window.location.hostname
```

Comment:
@haraldschilly -- good point, there is no window in node, so this code doesn't make any sense.